truffle/com.oracle.truffle.api.instrumentation/src/com/oracle/truffle/api/instrumentation/InstrumentationHandler.java

```diff
@@ -536,17 +538,18 @@ protected void visitInstrumented(Node node, SourceSection section) {
     }
 
     /**
-     * Instrumenter implementation for {@link TruffleInstrument}.
+     * Instrumenter implementation for use by {@linkplain TruffleInstrument external clients} of
+     * instrumentation.
      */
-    final class InstrumentationInstrumenter extends AbstractInstrumenter {
+    final class ClientInstrumenter extends AbstractInstrumenter {
```

Comment:
Any opinions on an earlier thought: renaming both internal subclasses to` InstrumentClientInstrumenter` and `Language ClientInstrumenter`